packages/create-app/src/createApp.ts

```diff
@@ -109,13 +109,16 @@ export default async (opts: OptionValues): Promise<void> => {
       await moveAppTask(tempDir, appDir, answers.name);
     }
 
-    if (gitConfig?.name && gitConfig?.email) {
-      Task.section('Initializing git repository');
-      await initGitRepository(appDir);
+    if (gitConfig) {
+      if (await tryInitGitRepository(appDir)) {
+        // Since we don't know whether we were able to init git before we
+        // try, we can't track the actual task execution
+        Task.forItem('init', 'git repository', async () => {});
```

Comment:
Is this because we track stderr or something? Because at the surface it sounds like otherwise it'd be better to just reword the task to be "Ensuring that a git repository is in place" or something, putting the work inside it, and it dealing with both cases properly.

Reply:
The messaging need to be pretty terse, so went with this. It's being shown in the "Moving to final location" section